app/src/main/java/com/example/cinemapp/util/DateUtil.kt

```diff
@@ -0,0 +1,19 @@
+package com.example.cinemapp.util
+
+import java.time.LocalDate
+import java.time.format.DateTimeFormatter
+import java.util.Calendar
+import java.util.Date
+
+fun LocalDate.setPattern(): String{
+    return DateTimeFormatter.ofPattern(DATE_PATTERN).format(this)
+}
+
+fun LocalDate?.ageAndRangeUntil(deathday: LocalDate?): String {
```

Comment:
Improve function name since body is not self explanatory...also, use camel case while naming parameters

Reply:
Camel case was used for the parameter, deathday is one word like birthday.